05-16 23:12:50 jdbc[3]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:120)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:133)
	at prerna.poi.main.AbstractFileReader.createNewRDBMSEngine(AbstractFileReader.java:386)
	at prerna.poi.main.AbstractFileReader.openRdbmsEngineWithoutConnection(AbstractFileReader.java:367)
	at prerna.poi.main.RDBMSReader.importFileWithOutConnection(RDBMSReader.java:127)
	at prerna.ui.components.ImportDataProcessor.processCreateNew(ImportDataProcessor.java:283)
	at prerna.ui.components.ImportDataProcessor.runProcessor(ImportDataProcessor.java:94)
	at prerna.upload.DatabaseUploader.uploadCSVFile(DatabaseUploader.java:308)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:137)
	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:296)
	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:250)
	at org.jboss.resteasy.core.ResourceLocatorInvoker.invokeOnTargetObject(ResourceLocatorInvoker.java:140)
	at org.jboss.resteasy.core.ResourceLocatorInvoker.invoke(ResourceLocatorInvoker.java:103)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:356)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:179)
	at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:220)
	at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:56)
	at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:51)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at prerna.web.conf.UserDBInsightFilter.doFilter(UserDBInsightFilter.java:80)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at prerna.web.conf.UserDBFilter.doFilter(UserDBFilter.java:112)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:504)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:950)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:421)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1074)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:611)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:314)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Unknown Source)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-185]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:174)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:171)
	at prerna.engine.impl.InsightsConverter.generateNewInsightsRDBMS(InsightsConverter.java:94)
	at prerna.engine.impl.AbstractEngine.createInsights(AbstractEngine.java:258)
	at prerna.ui.components.ImportDataProcessor.processCreateNew(ImportDataProcessor.java:298)
	at prerna.ui.components.ImportDataProcessor.runProcessor(ImportDataProcessor.java:94)
	at prerna.upload.DatabaseUploader.uploadCSVFile(DatabaseUploader.java:308)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.jboss.resteasy.core.MethodInjectorImpl.invoke(MethodInjectorImpl.java:137)
	at org.jboss.resteasy.core.ResourceMethodInvoker.invokeOnTarget(ResourceMethodInvoker.java:296)
	at org.jboss.resteasy.core.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:250)
	at org.jboss.resteasy.core.ResourceLocatorInvoker.invokeOnTargetObject(ResourceLocatorInvoker.java:140)
	at org.jboss.resteasy.core.ResourceLocatorInvoker.invoke(ResourceLocatorInvoker.java:103)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:356)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:179)
	at org.jboss.resteasy.plugins.server.servlet.ServletContainerDispatcher.service(ServletContainerDispatcher.java:220)
	at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:56)
	at org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher.service(HttpServletDispatcher.java:51)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:727)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:303)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:52)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at prerna.web.conf.UserDBInsightFilter.doFilter(UserDBInsightFilter.java:80)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at prerna.web.conf.UserDBFilter.doFilter(UserDBFilter.java:112)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:241)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:208)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:220)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:122)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:504)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:170)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:103)
	at org.apache.catalina.valves.AccessLogValve.invoke(AccessLogValve.java:950)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:116)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:421)
	at org.apache.coyote.http11.AbstractHttp11Processor.process(AbstractHttp11Processor.java:1074)
	at org.apache.coyote.AbstractProtocol$AbstractConnectionHandler.process(AbstractProtocol.java:611)
	at org.apache.tomcat.util.net.JIoEndpoint$SocketProcessor.run(JIoEndpoint.java:314)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 56 more
07-20 21:31:01 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2769)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:172)
	at prerna.engine.impl.AbstractEngine.openDB(AbstractEngine.java:200)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:107)
	at prerna.util.Utility.loadEngine(Utility.java:1226)
	at prerna.util.Utility.loadWebEngine(Utility.java:1115)
	at prerna.util.SMSSWebWatcher.loadNewDB(SMSSWebWatcher.java:91)
	at prerna.util.SMSSWebWatcher.loadExistingDB(SMSSWebWatcher.java:71)
	at prerna.util.SMSSWebWatcher.loadFirst(SMSSWebWatcher.java:157)
	at prerna.util.SMSSWebWatcher.run(SMSSWebWatcher.java:215)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 33 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 28 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(Unknown Source)
	at sun.nio.ch.SharedFileLockTable.add(Unknown Source)
	at sun.nio.ch.FileChannelImpl.tryLock(Unknown Source)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(Unknown Source)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 29 more
07-21 20:16:41 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2769)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:172)
	at prerna.engine.impl.AbstractEngine.openDB(AbstractEngine.java:200)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:107)
	at prerna.util.Utility.loadEngine(Utility.java:1227)
	at prerna.util.Utility.loadWebEngine(Utility.java:1116)
	at prerna.util.SMSSWebWatcher.loadNewDB(SMSSWebWatcher.java:91)
	at prerna.util.SMSSWebWatcher.loadExistingDB(SMSSWebWatcher.java:71)
	at prerna.util.SMSSWebWatcher.loadFirst(SMSSWebWatcher.java:157)
	at prerna.util.SMSSWebWatcher.run(SMSSWebWatcher.java:215)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 33 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 28 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(Unknown Source)
	at sun.nio.ch.SharedFileLockTable.add(Unknown Source)
	at sun.nio.ch.FileChannelImpl.tryLock(Unknown Source)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(Unknown Source)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 29 more
07-21 20:16:53 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2769)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:172)
	at prerna.engine.impl.AbstractEngine.openDB(AbstractEngine.java:200)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:107)
	at prerna.util.Utility.loadEngine(Utility.java:1227)
	at prerna.util.Utility.loadWebEngine(Utility.java:1116)
	at prerna.util.SMSSWebWatcher.loadNewDB(SMSSWebWatcher.java:91)
	at prerna.util.SMSSWebWatcher.loadExistingDB(SMSSWebWatcher.java:71)
	at prerna.util.SMSSWebWatcher.loadFirst(SMSSWebWatcher.java:157)
	at prerna.util.SMSSWebWatcher.run(SMSSWebWatcher.java:215)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 33 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 28 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(Unknown Source)
	at sun.nio.ch.SharedFileLockTable.add(Unknown Source)
	at sun.nio.ch.FileChannelImpl.tryLock(Unknown Source)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(Unknown Source)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 29 more
07-21 20:17:14 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2769)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:172)
	at prerna.engine.impl.AbstractEngine.openDB(AbstractEngine.java:200)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:107)
	at prerna.util.Utility.loadEngine(Utility.java:1227)
	at prerna.util.Utility.loadWebEngine(Utility.java:1116)
	at prerna.util.SMSSWebWatcher.loadNewDB(SMSSWebWatcher.java:91)
	at prerna.util.SMSSWebWatcher.loadExistingDB(SMSSWebWatcher.java:71)
	at prerna.util.SMSSWebWatcher.loadFirst(SMSSWebWatcher.java:157)
	at prerna.util.SMSSWebWatcher.run(SMSSWebWatcher.java:215)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 33 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 28 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(Unknown Source)
	at sun.nio.ch.SharedFileLockTable.add(Unknown Source)
	at sun.nio.ch.FileChannelImpl.tryLock(Unknown Source)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(Unknown Source)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 29 more
08-19 13:22:32 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2769)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:172)
	at prerna.engine.impl.AbstractEngine.openDB(AbstractEngine.java:200)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:107)
	at prerna.util.Utility.loadEngine(Utility.java:1227)
	at prerna.util.Utility.loadWebEngine(Utility.java:1116)
	at prerna.util.SMSSWebWatcher.loadNewDB(SMSSWebWatcher.java:91)
	at prerna.util.SMSSWebWatcher.loadExistingDB(SMSSWebWatcher.java:71)
	at prerna.util.SMSSWebWatcher.loadFirst(SMSSWebWatcher.java:157)
	at prerna.util.SMSSWebWatcher.run(SMSSWebWatcher.java:215)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 33 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 28 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(Unknown Source)
	at sun.nio.ch.SharedFileLockTable.add(Unknown Source)
	at sun.nio.ch.FileChannelImpl.tryLock(Unknown Source)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(Unknown Source)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 29 more
10-25 11:41:52 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2769)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:169)
	at prerna.engine.impl.AbstractEngine.openDB(AbstractEngine.java:195)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:105)
	at prerna.util.Utility.loadEngine(Utility.java:1059)
	at prerna.util.Utility.loadWebEngine(Utility.java:633)
	at prerna.util.SMSSWebWatcher.loadNewDB(SMSSWebWatcher.java:91)
	at prerna.util.SMSSWebWatcher.loadExistingDB(SMSSWebWatcher.java:71)
	at prerna.util.SMSSWebWatcher.loadFirst(SMSSWebWatcher.java:157)
	at prerna.util.SMSSWebWatcher.run(SMSSWebWatcher.java:215)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 33 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 28 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(Unknown Source)
	at sun.nio.ch.SharedFileLockTable.add(Unknown Source)
	at sun.nio.ch.FileChannelImpl.tryLock(Unknown Source)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(Unknown Source)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 29 more
10-25 11:42:05 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2769)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:166)
	at prerna.engine.impl.AbstractEngine.openDB(AbstractEngine.java:183)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:104)
	at prerna.util.Utility.loadEngine(Utility.java:1059)
	at prerna.util.Utility.loadWebEngine(Utility.java:633)
	at prerna.util.SMSSWebWatcher.loadNewDB(SMSSWebWatcher.java:91)
	at prerna.util.SMSSWebWatcher.loadExistingDB(SMSSWebWatcher.java:71)
	at prerna.util.SMSSWebWatcher.loadFirst(SMSSWebWatcher.java:157)
	at prerna.util.SMSSWebWatcher.run(SMSSWebWatcher.java:215)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 33 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 28 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(Unknown Source)
	at sun.nio.ch.SharedFileLockTable.add(Unknown Source)
	at sun.nio.ch.FileChannelImpl.tryLock(Unknown Source)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(Unknown Source)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 29 more
10-25 11:42:19 database: flush
org.h2.message.DbException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at org.h2.mvstore.MVStore.panic(MVStore.java:377)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:360)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2769)
	at org.h2.mvstore.db.MVTableEngine$Store.<init>(MVTableEngine.java:162)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:98)
	at org.h2.engine.Database.getPageStore(Database.java:2379)
	at org.h2.engine.Database.open(Database.java:666)
	at org.h2.engine.Database.openDatabase(Database.java:266)
	at org.h2.engine.Database.<init>(Database.java:260)
	at org.h2.engine.Engine.openSession(Engine.java:60)
	at org.h2.engine.Engine.openSession(Engine.java:167)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:145)
	at org.h2.engine.Engine.createSession(Engine.java:128)
	at org.h2.engine.Engine.createSession(Engine.java:26)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:347)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:108)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:92)
	at org.h2.Driver.connect(Driver.java:72)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:166)
	at prerna.engine.impl.AbstractEngine.openDB(AbstractEngine.java:183)
	at prerna.engine.impl.rdbms.RDBMSNativeEngine.openDB(RDBMSNativeEngine.java:104)
	at prerna.util.Utility.loadEngine(Utility.java:1059)
	at prerna.util.Utility.loadWebEngine(Utility.java:633)
	at prerna.util.SMSSWebWatcher.loadNewDB(SMSSWebWatcher.java:91)
	at prerna.util.SMSSWebWatcher.loadExistingDB(SMSSWebWatcher.java:71)
	at prerna.util.SMSSWebWatcher.loadFirst(SMSSWebWatcher.java:157)
	at prerna.util.SMSSWebWatcher.run(SMSSWebWatcher.java:215)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]" [50000-185]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 33 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/alsherman/workspace/SemossDev/db/Gene_to_Protein/insights_database.mv.db [1.4.185/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:165)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:346)
	... 28 more
Caused by: java.nio.channels.OverlappingFileLockException
	at sun.nio.ch.SharedFileLockTable.checkList(Unknown Source)
	at sun.nio.ch.SharedFileLockTable.add(Unknown Source)
	at sun.nio.ch.FileChannelImpl.tryLock(Unknown Source)
	at org.h2.store.fs.FileNio.tryLock(FilePathNio.java:121)
	at org.h2.mvstore.cache.FilePathCache$FileCache.tryLock(FilePathCache.java:152)
	at java.nio.channels.FileChannel.tryLock(Unknown Source)
	at org.h2.mvstore.FileStore.open(FileStore.java:162)
	... 29 more
